kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/utils/MetadataColumnTestUtils.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.delta.kernel.defaults.utils
+
+import io.delta.kernel.types.{FieldMetadata, LongType, MetadataColumnType, StructField}
+
+trait MetadataColumnTestUtils {
+  val ROW_INDEX = new StructField(
```

Comment:
Does this intentionally not  use `createmetadataColumn`

Reply:
No, that was an oversight. I'll fix this if we agree on this option.

Reply:
Done